modules/using-certificates-with-data-science-pipelines.adoc

```diff
@@ -0,0 +1,85 @@
+:_module-type: PROCEDURE
+
+[id='using-certificates-with-data-science-pipelines_{context}']
+= Using certificates with data science pipelines 
+
+== Providing a CA bundle only for data science pipelines
+
+Perform the following steps to provide a Certificate Authority (CA) bundle just for data science pipelines.
+
+.Procedure
+. Create a ConfigMap with the required bundle in the same data science project or namespace as the target data science pipeline:
```

Comment:
Are they doing this is OpenShift > Workloads > ConfigMaps or somewhere else?

Reply:
They are - have updated to add a step to log in to OCP, and indicate where to create the configmap.